### DISCUSSION TITLE
Gameplay buttons working in the escape menu

### DISCUSSION THREAD

**Discussion by xSxnic** [ppy/osu#31048] [Ideas] [answered]
pretty self explanatory being able to press "z" or "x" to trigger the exit, restart, and resume in the escape menu just like stable. (this has been a lazer killer for me tbh). also tell me if i am blind and there is a toggle or something. Thanks

**Comment by Joehuu** [comment, marked as answer]
This is already tracked as an issue in https://github.com/ppy/osu/issues/3551.
